app/helpers/ncr/work_orders_helper.rb

```diff
@@ -1,9 +1,8 @@
 module Ncr
   module WorkOrdersHelper
     def approver_options
-      # @todo should this list be limited by client/something else?
       # @todo is there a better order? maybe by current_user's use?
-      User.order(:email_address).pluck(:email_address)
+      User.where(client_slug: 'ncr').order(:email_address).pluck(:email_address)
```

Comment:
Seems OK to keep as-is.

To explain my thinking: If it were me, I would write a unit test for this because it is business-specific. It's a query on the `User` class. It happens to be used by a helper, but the query itself is an operation on `User` so it would be nice to just call `User.ordered_by_email_address` and know that the underlying logic is covered by a unit test.

Again, totally fine as-is, that's just the thinking on how I would handle it.